File: exercicios/R/1176/1176.r
input = file('stdin','r')

t = as.integer(readLines(input,n=1))

for(i in 1:t){
    n = as.integer(readLines(input,n=1))

    f <- numeric(n)

    f[1] = 0
    f[2] = 1

    if(n>1){
        for(j in 2:n+1){
            f[j] = f[j-2]+f[j-1]
        }
        write(sprintf("Fib(%d) = %.0f",j-1,f[j]),'')
    }else if(n<=1){
        write(sprintf("Fib(%d) = %d",n,n),'')
    }
}

File: exercicios/JavaScript/1176/1176.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var t = parseInt(lines.shift());


for(let i=0; i<t;i++){
    var x = parseInt(lines.shift());
    var f = [x+t];
    f[0] = 0;
    f[1] = 1;
                
    if(x>1){
        for(let j=2;j<=x+1;j++){ 
            f[j] = f[j-2]+f[j-1];
            }
                console.log("Fib(%d) = %d",x,f[x]);
            }else if(x<=1){
                console.log("Fib(%d) = %d",x,x);
            }
        }
